spec/features/cart/user_can_view_items_in_cart_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature "When a customer visits the home page and wants to view their cart" do
+
+  let!(:pants) { create(:item, title: "Rustic Wooden Pants", price_in_cents: 11_11) }
+  let!(:computer) { create(:item, title: "Synergistic Silk Computer", price_in_cents: 22_22) }
+  let!(:spoon) { create(:item, title: "Four Dollar Wooden Spoon", price_in_cents: 33_33) }
+
+  before do
```

Comment:
Can we also add a sad path test for what happens when they try to view the cart when the cart is empty?